tests/foreman/ui/test_remoteexecution.py

```diff
@@ -404,6 +428,12 @@ def test_positive_run_default_job_template(self):
                     job_template='Run Command - SSH Default',
                     options_list=[{'name': 'command', 'value': 'ls'}]
                 )
+                # get job invocation id from the current url
```

Comment:
Please, explain why we need these blocks and what impact on the test result they have. I see that we are getting some data, but do not assign results of that getter to any variable